### ISSUE TITLE
Turning off Section Collapse hides section headings in View mode

### ISSUE THREAD

**Issue by ajw625** [orbeon/orbeon-forms#2675]
I added the following properties to disable the expand/collapse feature on section headings. I'm not sure where I got these. It might have been from this discussion post: http://discuss.orbeon.com/Section-title-behavior-td4656761.html. 

    <property as="xs:boolean" name="oxf.fr.detail.noscript.section.collapse.*.*" value="false"/>
    <property as="xs:boolean" name="oxf.fr.detail.ajax.section.collapse.*.*" value="false"/>

It works as expected in `edit` mode, but in `view` mode the headers containing the section names are not visible.

I tested this with a clean Orbeon 4.10 install.

#### View mode normally:
![image](https://cloud.githubusercontent.com/assets/9143532/14216940/4a047fb6-f800-11e5-9118-b31b91f9fa0b.png)

#### View mode with the above properties:
![image](https://cloud.githubusercontent.com/assets/9143532/14216980/7c2bc1d4-f800-11e5-9bbe-730adbaf95a2.png)

**Comment by ebruchez** [comment]
Thanks for the report. The good news is that it works now, so it will work with the upcoming Orbeon Forms 4.11. See screenshot.

By the way,  the [property names](http://doc.orbeon.com/configuration/properties/form-runner.html#section-collapsing) have been updated to:

```xml
<property
    as="xs:boolean"
    name="oxf.xforms.xbl.fr.section.collapsible.*.*"
    value="false"/>
```

and:

```xml
<property
    as="xs:boolean"
    name="oxf.xforms.xbl.fr.section.noscript.collapsible.*.*"
    value="false"/>
```

(The old names are still supported for backward compatibility.)

![collapsible](https://cloud.githubusercontent.com/assets/105769/14222266/899ba168-f823-11e5-9282-f113d1901b05.png)

**Comment by ebruchez** [comment]
Closing as fixed in 4.11.

**Comment by ajw625** [comment]
Thanks, Erik!
